### PR TITLE
modified succesfully

### PR DESCRIPTION
**Pull request by sivaazure1999**
please check code

### CODE REVIEW

File: ansible1.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 welcome to ansible
 
-ansible playbook for 
\ No newline at end of file
+ansible playbook for    
+
+ygdsh
```
Comment (siva77sai) [inline ansible1.txt:5]: **this is not correct**

File: terraform1.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-welcome to terraform
\ No newline at end of file
+welcome to terraform
+
+hdfjhds
```
Comment (siva77sai) [inline terraform1.txt:3]: this line error

**Review by siva77sai — COMMENTED** [review]
please check correctly

File: ansible1.txt (hunk 1 above)
Comment (sivaazure1999) [reply ansible1.txt:5]: i can change the above code

File: terraform1.txt (hunk 2 above)
Comment (sivaazure1999) [reply terraform1.txt:3]: changed that line

**Pull request merged**